controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -280,7 +282,42 @@ void wfa_ca::handle_wfa_ca_message(
         break;
     }
     case eWfaCaCommand::DEV_GET_PARAMETER: {
-        // TODO
+
+        /* 
+        * This command retrieves the requested parameter information and sends the information in the response.
+        *
+        * Parameters:
+        * 
+        * Param Name    | Values                        | Description
+        * ----------------------------------------------------------------------------------------
+        * "program"     | string (MAP)                  | Program name
+        * "ruid"        | string                        | A radio Unique ID
+        * "SSID"        | string                        | The SSID that fronthaul AP is configured. When SSID is not specified for the parameter
+                                                          "macaddr", it implies that device should return backhaul STA's MAC address. SSID is also
+                                                          not required for the parameter "ALid"
+        * 
+        * Return Values: 
```

Comment:
Missing param name "parameter" on the table.
As well the description exceeds 100 notes. Please fix this on other tables you added. I look better when open the code in github.

Reply:
removed all comments since they violate license